README.md

```diff
@@ -12,24 +12,21 @@ To use the application sensor, use convivaAppTracker.js file from this repositor
 
 #### Add the following script to the HTML head section of your website:
 ```js
-<script type="text/javascript" async=1>
-;
-(function(p, l, o, w, i, n, g) {
+<script type="text/javascript" async>
+
+(function(p, i) {
     if (!p[i]) {
         p.GlobalSnowplowNamespace = p.GlobalSnowplowNamespace || [];
         p.GlobalSnowplowNamespace.push(i);
         p[i] = function() {
             (p[i].q = p[i].q || []).push(arguments)
         };
         p[i].q = p[i].q || [];
-        n = l.createElement(o);
```

Comment:
here we are creating script tag providing `convivaAppTracker.js` url as src and adding that script on the top of all script tags ,
to achieve new network request tracking feature `convivaAppTracker.js` has to be included before any another package with current implementation being async this was not working out so changed slight instructions.